### ISSUE TITLE
Autohost feature?

### ISSUE THREAD

**Issue by hotwer** [Josko/aura-bot#80]
This is not a complain, but it would be awesome having it implemented.

I was already able to sucessfully launch the bot on battle.net, but didn't tried to play a game with it yet.

Do you think implementing such a feature would be that hard?

**Comment by hotwer** [comment]
By the way, has josko left? He has 2 pull requets almost a month old and didn't reply to any (both seem nice).

If he really dropped I'm willing to go forward with his project.

**Comment by HelloWorldAndEveryone** [comment]
Idk, but it's normal. Many developers leave their projects because they have no motivation for doing it, but usually they continue making after few months/years(me too)
But you always can fork it and make your own changes, and maybe if he will back he will accept pull requests with your changes.

If you really wanna make this bot better, firstly realize the autohosting like in ghostbot, really important function, I guess :)

**Comment by hotwer** [comment]
Yes, and that's why I care for a lot it being developed, at least once a week to check for peoples pull requests. More a project is contributed, the better the final product will be, and if the main contributor is just away (even if not coding, he could at least be managing peoples entries).

This desmotivates everyone. Yes, I can always fork, but doing so I will not benefit from others peoples implementations.

**Comment by phongse** [comment]
No, autohosting was removed for a reason and will most likely never be implemented again.
Read [this](https://github.com/Josko/aura-bot/issues/57#issuecomment-355355299).

And I can't tell you if Josko dropped this project but it is very unlikely that he did. Probably other priorities right now.

**Comment by hotwer** [comment]
I see, although I understand the reason why it was decided to not be implemented, I disagree about this decision hugely, but it can't be helped. If that's aura authors decision, god knows why, I really don't understand any reason why not having such feature implemented. It just makes the software better. 

Anyway, if this is the final decision, you can close this if you want. I really don't undestant what kind of demand Aura is trying to fill, it seems to me like a waste. I thought the features were dropped in favor of a better core, and as I see, sadly, this is not the case.

**Comment by HelloWorldAndEveryone** [comment]
So, are you trying to implement this? I'm waiting *_*

**Comment by hotwer** [comment]
Sorry, no, I didn't touch in this matter myself.
